document/core/syntax/modules.rst

```diff
@@ -241,10 +240,10 @@ The |MEVENTS| component of a module defines a vector of *events* with the follow
 
 .. math::
    \begin{array}{llll}
-   \production{event} & \event &::=& \{ \EVATTRIBUTE~\EXCEPTION, \EVTYPE~\typeidx \} \\
+   \production{event} & \event &::=& \{ \EVATTR~\EXCEPTION, \EVTYPE~\typeidx \} \\
    \end{array}
 
-The event's :ref:`attribute <syntax-attribute>` may only specify the attribute |EXCEPTION|, meaning that the event is an exception.
+The :ref:`event attribute <syntax-eventattr>` may only specify the attribute |EXCEPTION|, meaning that the event is an exception.
```

Comment:
As long as it basically carries no information (there is only one possible value), I'd completely remove event attributes from the abstract syntax for the time being. For future extensibility it suffices to have the zero byte in the binary format.

Reply:
Sure, no problem at all to completely remove "event" everywhere (except the binary format byte you mentioned). I just thought that in your [first comment ](https://github.com/WebAssembly/exception-handling/pull/121#pullrequestreview-435019490) you suggested to do this in a different PR, that is the reason why I didn't do that yet. 

The next commit will remove all occurrences of event. 

Just to make sure I understand correctly, I should also rename "event section" to "exception section", so really leave no trace of "event" except in that binary format bit, correct?

Reply:
Ah, okay, I wasn't sure what the plan was, since I think there were some changes already. Anyway, I think we're on the same page, so I'm fine either way.

Yes, I'd rename the event section as well (for now).